packages/common/http/src/transfer_cache.ts

```diff
@@ -24,8 +24,12 @@ interface TransferHttpResponse {
   responseType?: HttpRequest<unknown>['responseType'];
 }
 
-const CACHE_STATE = new InjectionToken<{isCacheActive: boolean}>(
-    ngDevMode ? 'HTTP_TRANSFER_STATE_CACHE_STATE' : '');
+type CacheOptions = {
+  isCacheActive: boolean;
+}&HttpTransferCacheOptions&{includePostRequests?: boolean};
```

Comment:
Consider using an interface

```suggestion
interface CacheOptions extends HttpTransferCacheOptions {
  isCacheActive: boolean;
  includePostRequests?: boolean;
}
```